hooks/survey/tests/UseSurvey.test.ts

```diff
@@ -1,20 +1,21 @@
 import { getSurveyMessageMock } from '@/__tests__/mock/survey/types/surveyMessageMock';
 import { renderHook, RenderHookResult } from '@testing-library/react';
 import { act } from '@testing-library/react-hooks';
-import { ChoiceVO, QuestionMessage, QuestionVO } from '@/types/SurveyMessage';
+import { Question } from '@/types/Survey';
+import { ChoiceUI, QuestionMessage, QuestionUI } from '@/types/SurveyMessage';
 import useQuestion, { UseQuestion } from '../useQuestion';
 import { data } from './surveyMock.json';
 
 // TODO : 설문이 완료되었을 때의 동작 테스트 필요
 describe('Given : UseSurvey ', () => {
   const surveyMessageMock: QuestionMessage[] = data.map((_) => {
-    return getSurveyMessageMock(_ as QuestionVO);
+    return getSurveyMessageMock(_ as QuestionUI);
   });
 
   let renderResult: RenderHookResult<UseQuestion, null>['result'];
   beforeEach(() => {
     const { result } = renderHook(() =>
-      useQuestion({ surveys: data as QuestionVO[] }),
+      useQuestion({ surveys: data as unknown as Question[] }),
```

Comment:
as unknown 은 의도된건가요?
```suggestion
      useQuestion({ surveys: data as Question[] }),
```

Reply:
넵 의도된겁니다. 그리고 이 부분은 여쭤보고 싶었습니다.
`Question`과 `Choice`는 아래의 사진처럼 모델링이 되어있습니다.

<img width="604" alt="image" src="https://github.com/f-lab-edu/ClothesGPT/assets/53801395/7aba3f69-1e1e-4638-8122-a0205d2537c9">

`Question`에는 여러 개의 `Choice`가 있고 `Choice`에는 한 개의 `Question`을 갖고 있으니 1 - n 관계이며 순환 참조하고 있는 상태입니다.
그래서 이와 같은 json mock data를 만드는게 불가능합니다.

<img width="441" alt="image" src="https://github.com/f-lab-edu/ClothesGPT/assets/53801395/daeb9f8d-330b-41a6-b780-d21bd84d3dce">

그래서 임시방편으로 as unknown으로 처리하여 타입 에러가 안나게 했지만 좋아보이진 않습니다..
`Question`과 `Choice`가 서로 순환 참조되게 만든게 문제인거같은데 이를 해결할 방법이 딱히 떠오르지 않네요.

Reply:
model 이랑 TypeScript Type 이랑 꼭 1:1 로 매칭되어 순환 참조 할 필요는 없을것 같습니다.
되도록이면 순환을 피하게 TypeScript Type 을 지정하는게 좋구요.
일단 이건 그대로 둬도 무방할것 같습니다.